lib/cryptoexchange/exchanges/bitshares_assets/services/market.rb

```diff
@@ -0,0 +1,35 @@
+module Cryptoexchange::Exchanges
+  module BitsharesAssets
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch(market_pair)
+          output = super(ticker_url)
+          adapt(output, market_pair)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::BitsharesAssets::Market::API_URL}/asset/markets?asset=BTS"
```

Comment:
we only want `BTS` ?

Reply:
BTS only as starter, otherwise too many pairs